src/app/packages/menu-items/layers-manager/layers-manager/layers-manager.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { TreeNode } from 'angular-tree-component';
+import { ILayerTreeNode } from '../models/layer-tree-node';
+@Component({
+  selector: 'app-layer-managers',
+  templateUrl: './layers-manager.component.html',
+  styleUrls: ['layers-manager.component.scss']
+})
+export class LayersManagerComponent implements OnInit {
+
+  private nodes: ILayerTreeNode[];
+  constructor() { }
+
+  ngOnInit() {
+
+    let rootNode: ILayerTreeNode = {
```

Comment:
This is all an hard coded mock data.
Please integrate this with mock service and not with hardcoded mock.
Add a develop configuration and producrtion configuration.